app/summarize.js

```diff
@@ -1,26 +1,40 @@
+var _ = require('underscore');
+
 module.exports = function(event) {
   var errors = [];
+  // Deep-clone the event plan
   var summary = JSON.parse(JSON.stringify(event.plan));
+
   summary.forEach(function(f, i){
+    // Plan and snapshot offsets may not match, get the correct key
+    var fkey = _.findKey(event.snapshot, function (feature) {
```

Comment:
I've added ```
 tags=smart
 show_skipped=no
``` to my local behave.ini so that I can test *only* the oauth part of the process, makes things run faster. In this case, the snapshot only contains the tests I've run.